brainscore/benchmarks/freemanziemba2013.py

```diff
@@ -40,13 +42,48 @@ def MovshonFreemanZiemba2013V1RDM():
                                            ceiler=RDMConsistency())
 
 
+def MovshonFreemanZiemba2013V2PLS():
+    return _MovshonFreemanZiemba2013Region('V2', identifier_metric_suffix='pls',
+                                           similarity_metric=CrossRegressedCorrelation(
+                                               regression=pls_regression(), correlation=pearsonr_correlation(),
+                                               crossvalidation_kwargs=dict(stratification_coord='texture_type')),
+                                           ceiler=InternalConsistency())
+
+
 def MovshonFreemanZiemba2013V2RDM():
     return _MovshonFreemanZiemba2013Region('V2', identifier_metric_suffix='rdm',
                                            similarity_metric=RDMCrossValidated(
                                                crossvalidation_kwargs=dict(stratification_coord='texture_type')),
                                            ceiler=RDMConsistency())
 
 
+# V1 benchmarks separated by access
+def _MovshonFreemanZiemba2013RegionAccess(region, identifier_metric_suffix, similarity_metric, ceiler, access='private'):
+    assembly_repetition = LazyLoad(lambda region=region: load_assembly(False, region=region))
+    assembly = LazyLoad(lambda region=region: load_assembly(True, region=region, access=access))
+    return NeuralBenchmark(identifier=f'movshon.FreemanZiemba2013.{access}.{region}-{identifier_metric_suffix}', version=2,
+                           assembly=assembly, similarity_metric=similarity_metric, visual_degrees=VISUAL_DEGREES,
+                           ceiling_func=lambda: ceiler(assembly_repetition),
+                           parent=region,
+                           paper_link='https://www.nature.com/articles/nn.3402')
+
+
+def MovshonFreemanZiemba2013V1PLSPrivate():
+    return _MovshonFreemanZiemba2013RegionAccess('V1', identifier_metric_suffix='pls',
+                                                 similarity_metric=CrossRegressedCorrelation(
+                                                 regression=pls_regression(), correlation=pearsonr_correlation(),
+                                                 crossvalidation_kwargs=dict(stratification_coord='texture_type')),
+                                                 ceiler=InternalConsistency(), access='private')
+
+
+def MovshonFreemanZiemba2013V1PLSPublic():
+    return _MovshonFreemanZiemba2013RegionAccess('V1', identifier_metric_suffix='pls',
+                                                 similarity_metric=CrossRegressedCorrelation(
+                                                 regression=pls_regression(), correlation=pearsonr_correlation(),
+                                                 crossvalidation_kwargs=dict(stratification_coord='texture_type')),
+                                                 ceiler=InternalConsistency(), access='public')
+
+
```

Comment:
these should not be here. Public benchmarks are in https://github.com/brain-score/brain-score/blob/master/brainscore/public_benchmarks.py